### PR TITLE
feat(IDX): switch to plaintext file

### PR DESCRIPTION
**Pull request by cgundy**

### CODE REVIEW

File: reusable_workflows/repo_policies/bot_checks/check_bot_approved_files.py
Comment (basvandijk) [inline reusable_workflows/repo_policies/bot_checks/check_bot_approved_files.py:52]: Why do we actually want to error in case there are no approved files? What if we temporarily want to forbid any bot from changing any files by just emptying that file?

**Review by basvandijk — APPROVED** [review]
Nice!

File: reusable_workflows/repo_policies/bot_checks/check_bot_approved_files.py
Comment (cgundy) [reply reusable_workflows/repo_policies/bot_checks/check_bot_approved_files.py:52]: Ah well I wanted to make the error as informative as possible in case someone made a mistake in setting up the config, but I suppose I could also just let it fail in the next step where it checks if the changed files are within the list of approved files.

Comment (basvandijk) [reply reusable_workflows/repo_policies/bot_checks/check_bot_approved_files.py:52]: Yes makes sense. Allowing empty files (or files with only comments) should be allowed I think just like it's allowed for CODEOWNERS.

**Pull request merged**
